src/capabilities.js

```diff
@@ -135,7 +187,9 @@ export class Capabilities {
       if (authCoreId === this.#projectCreatorAuthCoreId) {
         return CREATOR_CAPABILITIES
       } else {
-        return DEFAULT_CAPABILITIES[BLOCKED_ROLE_ID]
+        // When no role assignment exists, e.g. a newly added device which has
+        // not yet synced role records.
+        return NO_ROLE_CAPABILITIES
```

Comment:
looking at this just a few lines above:

```ts
const authCoreId = await this.#coreOwnership.getCoreId(deviceId, 'auth')
```

do you have to wrap that in a try/catch, and return this no role capabilities in the case of the catch? with https://github.com/digidem/mapeo-core-next/pull/317, that call may throw an error if the document isn't found.

Basically, should this block look like this?

```ts
      try {
        const authCoreId = await this.#coreOwnership.getCoreId(deviceId, 'auth')
        if (authCoreId === this.#projectCreatorAuthCoreId) {
          return CREATOR_CAPABILITIES
        } else {
          // When no role assignment exists, e.g. a newly added device which has
          // not yet synced role records.
          return NO_ROLE_CAPABILITIES
        }
      } catch (err) {
        // When no role assignment exists, e.g. a newly added device which has
        // not yet synced role records.
        return NO_ROLE_CAPABILITIES
      }
```